src/context.ts

```diff
@@ -9,7 +9,7 @@ import {
 import { EqualityChecker, UseStore } from 'zustand'
 import { State, StateSelector } from './vanilla'
 
-interface UseStoreData<T extends State> {
+export interface UseStoreData<T extends State> {
```

Comment:
```suggestion
export interface UseStore<T extends State> {
```
`Data` looks strange to me. Is this too confusing?

Reply:
On the other hand, `UseStore` is already declared in the main file and it also contains the store API
```ts
export interface UseStore<T extends State> {
  (): T
  <U>(selector: StateSelector<T, U>, equalityFn?: EqualityChecker<U>): U
  setState: SetState<T>
  getState: GetState<T>
  subscribe: Subscribe<T>
  destroy: Destroy
}
```

I will be fine with renaming it to anything that you would like to, but just wanted to bring this up.

Reply:
Yeah, that's why I wondered if it's too confusing.
How about `UseStoreInContext`? This is for TS and users wouldn't typically write it directly, right?

Reply:
Or, `UseStoreWithoutApi`

Reply:
I like your first suggestion better I think, but to make it a little more readable how about `UseContextStore`?

Reply:
I'm ok with it.